source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,315 @@
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve" valid="feasible"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/foliadesc-04.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                        <p>
+                           <figure>
+                              <head>Positioning and sizing of a patch</head>
+                              <graphic url="../images/ExampleImages/foliadesc-04.png"/>
+                           </figure>
+                        </p>
+                        <p>The example above describes a bifolium where a patch is glued to the inner right side.</p>
+                     </div>
+                     <div xml:id="cutouts" type="div4">
+                        <head>Cutouts</head>
+                        <p>
+                           <specList>
+                              <specDesc key="cutout"/>
+                           </specList>
+                        </p>
+                        <p>Cutouts are treated almost similarly as <ptr target="#patches"/>. The most relevant attributes are:</p>
+                        <p>
+                           <specList>
+                              <specDesc key="cutout" atts="removed.from"/>
+                              <specDesc key="cutout" atts="removed.by"/>
+                           </specList>
+                        </p>
+                        <p>The dimensions (<att>width</att>, <att>height</att>) of the parent element (e.g. <gi scheme="MEI">folium</gi>) indicate the size of the bounding box of the remaining part of the page. That is, if the complete lower half of a page has been cut, the <att>width</att> and <att>height</att> attributes describe the remaining upper half. If, in contrast, only the lower right quarter of the page has been cut, these attributes still indicate the size of the full page (assuming that the removed section was a regular rectangle).The dimensions (<att>width</att>, <att>heigh</att>) on cutout itself are only to be used when there is a "gap" in the manuscript that allows to specify the dimensions of that missing part. In this case, the bounding box dimensions are given, together with <att>x</att> and <att>y</att> to indicate the upper left point on the original page. If, however, the removed section is available by itself, then a corresponding <gi scheme="MEI">folium</gi> (or <gi scheme="MEI">bifolium</gi>) should be placed inside the cutout element, and should provide it's own dimensions using <att>width</att> and <att>height</att> there. In this case, <att>width</att> and <att>height</att> on cutout is expendable.</p>
```

Comment:
```suggestion
                        <p>The dimensions (<att>width</att>, <att>height</att>) of the parent element (e.g., <gi scheme="MEI">folium</gi>) indicate the size of the bounding box of the remaining part of the page. That is, if the complete lower half of a page has been cut, the <att>width</att> and <att>height</att> attributes describe the remaining upper half. If, in contrast, only the lower right quarter of the page has been cut, these attributes still indicate the size of the full page (assuming that the removed section was a regular rectangle).The dimensions (<att>width</att>, <att>heigh</att>) on cutout itself are only to be used when there is a "gap" in the manuscript that allows to specify the dimensions of that missing part. In this case, the bounding box dimensions are given, together with <att>x</att> and <att>y</att> to indicate the upper left point on the original page. If, however, the removed section is available by itself, then a corresponding <gi scheme="MEI">folium</gi> (or <gi scheme="MEI">bifolium</gi>) should be placed inside the cutout element, and should provide it's own dimensions using <att>width</att> and <att>height</att> there. In this case, <att>width</att> and <att>height</att> on cutout is expendable.</p>
```

Reply:
Added comma after e.g.

Reply:
thanks for this notice, I haven't read the texts I just moved.